package.json

```diff
@@ -234,6 +240,7 @@
     "sinon": "*",
     "source-map-support": "^0.5.12",
     "storybook-react-router": "^1.0.5",
+    "ts-jest": "^24.0.2",
```

Comment:
This should be covered by `babel-jest`, in combination with the `@babel/preset-typescript` (though I imagine this is just experiment code)